appeals/api/src/server/endpoints/documents/documents.routes.js

```diff
@@ -2,27 +2,34 @@ import { Router as createRouter } from 'express';
 import { asyncHandler } from '#middleware/async-handler.js';
 import { getAppealValidator } from '#endpoints/appeals/appeals.validators.js';
 import { checkAppealExistsAndAddToRequest } from '#endpoints/appeals/appeals.service.js';
-import { ensureCaseFolders, validateDocumentAndAddToRequest } from './documents.middleware.js';
+import { validateDocumentAndAddToRequest } from './documents.middleware.js';
+import {
+	getFolderIdValidator,
+	getDocumentIdValidator,
+	getDocumentValidator,
+	getDocumentsValidator
+} from './documents.validators.js';
 import * as controller from './documents.controller.js';
 
 const router = createRouter();
 
 router.get(
-	'/:appealId/document-locations',
+	'/:appealId/document-location/:folderId',
```

Comment:
It looks like this endpoint is for getting the document folders for an appeal? 

If so then it would probably be more consistent for this endpoint to be `/:appealId/document-folders/:folderId` otherwise the endpoint is suggesting that we're getting document locations by folder id and document locations could be folders but could also be something else.

Reply:
ok changed as suggested